pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = ["LICENSE", "src/viam/rpc/libviam_rust_utils.*"]
   typing-extensions = "^4.7.1"
   Pillow = "^10.0.0"
   protobuf = "^4.23.4"
-  numpy = ">=1.21"
+  numpy = { version = ">=1.21", optional = true }
```

Comment:
re-included this bit that was moved to dev dependencies because I wanted to follow the docs to make sure the extras flag works with PEP508/pip install stuff

source: https://python-poetry.org/docs/pyproject/#extras